scripts/irods/test/test_session_tokens.py

```diff
@@ -259,7 +259,7 @@ def test_groupadmins_fail_to_authenticate_when_trying_to_get_nonexpiring_session
 
 
 @unittest.skipIf(test.settings.TOPOLOGY_FROM_RESOURCE_SERVER, "Must configure catalog provider for these tests.")
-class test_delete_session_tokens(unittest.TestCase):
+class test_remove_session_tokens(unittest.TestCase):
```

Comment:
Just noticed that there's no issue numbers on the class or functions. Is that intentional?